Example/Auth/Sample/MainViewController.m

```diff
@@ -2493,6 +2500,43 @@ - (void)createUser {
   }];
 }
 
+/** @fn createUserAuthDataResult
+    @brief Creates a new user.
+ */
+- (void)createUserAuthDataResult {
+  [self showTextInputPromptWithMessage:@"Email:"
+                          keyboardType:UIKeyboardTypeEmailAddress
+                       completionBlock:^(BOOL userPressedOK, NSString *_Nullable email) {
+    if (!userPressedOK || !email.length) {
+      return;
+    }
+
+    [self showTextInputPromptWithMessage:@"Password:"
+                         completionBlock:^(BOOL userPressedOK, NSString *_Nullable password) {
+      if (!userPressedOK) {
```

Comment:
this conditional doesn't handle empty/null password strings.

Reply:
That's by design, we want to be able to test that the API implementation validation via the Sample App (during manual testing).